letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -1428,6 +1431,12 @@ UNLIKELY_EOF
 
     say "Installation succeeded."
   fi
+
+  if ["$INSTALL_ONLY" = 1 ]; then
```

Comment:
So I'm looking into why the letsencrypt-auto Travis test we run for PRs passed, but you need a space between `[` and `"$INSTALL_ONLY"`. If you run the following script:
```
#!/bin/sh

if ["$INSTALL_ONLY" = 1 ]; then
    exit 0
fi
```
You'll get output like:
```
./test.sh: line 3: [: =: unary operator expected
```

Reply:
Well that's mildly terrifying. We have `set -e` in `letsencrypt-auto` which means to exit if any commands fail. Apparently this comparison isn't considered a failure, but it never seems to be true. It just always prints a warning and skips the body of the `if` statement.

Since this is new functionality and everything else in `letsencrypt-auto` works the same (sometimes with new error output), our tests pass.

Reply:
my bad-- had it changed locally and forgot to commit. i'll open a separate issue for the Travis issue!